man/systemd.system-credentials.xml

```diff
@@ -138,18 +138,20 @@
       </varlistentry>
 
       <varlistentry>
-        <term><varname>network.netdev.*</varname></term>
+        <term><varname>network.conf.*</varname></term>
         <term><varname>network.link.*</varname></term>
+        <term><varname>network.netdev.*</varname></term>
         <term><varname>network.network.*</varname></term>
         <listitem>
           <para>Configures network devices. Read by
-          <citerefentry><refentrytitle>systemd-network-generator.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>. These
-          credentials directly translate to a matching <filename>*.netdev</filename>,
-          <filename>*.link</filename> or <filename>*.network</filename> file. Example: the contents of a
-          credential <filename>network.link.50-foobar</filename> will be copied into a file
-          <filename>50-foobar.link</filename>. See
-          <citerefentry><refentrytitle>systemd.netdev</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+          <citerefentry><refentrytitle>systemd-network-generator.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
+          These credentials directly translate to a matching <filename>.conf</filename>,
+          <filename>*.link</filename>, <filename>*.netdev</filename>, or <filename>*.network</filename> file.
```

Comment:
Hmm, I think `*` doesn't make much sense here, and for `.conf` you didn't add that. So remove it from the rest config types for consistency?